tests/gene-comparison-tool.spec.ts

```diff
@@ -27,4 +27,15 @@ test.describe('specific viewport block', () => {
     const dropdown = page.locator('#subCategory');
     await expect(dropdown).toHaveText('Targeted Selected Reaction Monitoring (SRM)');
   });
+
+  test('invalid gene results in a 404 redirect', async ({ page }) => {
```

Comment:
Does this test belong with the GCT tests or would it make sense to include this test in the `gene-details` test file, since the fix was made in the `gene-details` component?

Reply:
You're correct.  It should be in its own file.  I will also create the same test for gene-similar